README.md

```diff
@@ -479,6 +474,18 @@ A component that takes [`FormProps`](#formprops) and surrounds your entire form.
 A component that takes [`FormSpyProps`](#formspyprops) and can listen to form
 state from inside an optimized `<Form/>`.
 
+### `useField`
+
+The `useField` hook takes two parameters, the first is the name of the field, and the second is an optional object that looks just like [`FieldProps`](#fieldprops), except without the name. It returns an object just like [`FieldRenderProps`](#fieldrenderprops).
+
+`useField` is used internally inside `Field`.
+
+### `useFormState`
+
+The `useFormState` hook takes one optional parameter, which matches the exact shape of [`FormSpyProps`](#formspyprops). It returns a [`FormSpyRenderProps`](#formspyrenderprops).
+
+`useFormState` is used internally inside `FormSpy`.
```

Comment:
you are not renaming FormSpy after all? :p

Reply:
Sometimes the vociferous minority in a vote can sway the outcome. 😜